src/internal/item-container/index.tsx

```diff
@@ -392,7 +398,7 @@ function ItemContainerComponent(
             : null,
         }}
       >
-        {children}
+        {inTransition && !isActive ? childrenRef.current : children()}
```

Comment:
Could this condition and the `if (!inTransition) {` above be combined for easier readability of the different conditions/cases? (Unless there's some reason why `childrenRef.current` should not be updated when `inTransition === true && isActive === true`, but I can't immediately see why that would be?)

Reply:
Will do